qcore/srf.py

```diff
@@ -11,15 +11,19 @@
 
 from math import ceil, cos, floor, radians, sqrt, sin, degrees, atan
 from subprocess import Popen, PIPE
+import sys
 
 import numpy as np
 
 try:
     import alphashape
+    from alphashape import optimizealpha
+    import shapely
```

Comment:
I think shapely is common enough that it doesn't need to be treated separately.

Reply:
Agreed -